package.json

```diff
@@ -29,7 +29,10 @@
     "doc:resources": "cp CHANGELOG.md ./docs",
     "clean": "rm -rf renderedTemplates dist",
     "renderTemplates": "gulp --gulpfile ./gulpfiles/renderTemplatesTask.ts renderTemplates",
-    "updateApis": "gulp --gulpfile ./gulpfiles/downloadRamlsTask.ts updateApis",
+    "backupApis": "rm -rf updateApisTmp && mkdir updateApisTmp && mv apis updateApisTmp/oldApis",
```

Comment:
prefer backup dir name as "updateApisBackup". Users exactly what this dir is for..

Reply:
This directory isn't just for the backups, it's for all of the files generated by `updateApis`. The backups are stored in a directory within this directory.